.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: "Release npm package"
+on:
+  schedule:
+  - cron: "0 0 1 * *"
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+      container: docker:20
+      
+      services:
+        docker:
+          image: docker:20-dind
+          env:
+            DOCKER_DRIVER: overlay2
+
+      steps:
+        - name: semantic-release
+        - uses: actions/checkout@v2
+        - uses: ./.github/workflows/composite-action
+        - run: |
```

Comment:
Replace the run with the https://github.com/marketplace/actions/action-for-semantic-release release action